src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -384,6 +395,124 @@ type Currency {
   digits: Int!
 }
 
+"Stoptime represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop."
+type DatedStopTime {
+  "Scheduled arrival time to the stop and a realtime estimate, if one exists."
+  arrival: ArrivalDepartureTime
```

Comment:
Do we need to take flex ranges into account here?

Reply:
Maybe, do you have any suggestion on how?

Reply:
We decided to make an union for the trip instead of the stop time so we can later on add types for flex and frequency based trips.